infer/src/IR/tenv.ml

```diff
@@ -92,6 +92,29 @@ let proc_extract_declaring_class_typ tenv pname_java =
 let proc_extract_return_typ tenv pname_java =
   lookup_java_typ_from_string tenv (Procname.java_get_return_type pname_java)
 
+(** Get method that is being overriden by java_pname (if any) **)
```

Comment:
Can you run ocp-indent on this file to use our formatting style? This should do it:

```
opam install ocp-indent
ocp-indent -i <file>
```